### ISSUE TITLE
Issues with large titles 

### ISSUE THREAD

**Issue by jethmalani** [jcavar/refresher#38]
Just a heads up, a lot of stuff gets messed up when large titles are used in iOS 11.

**Comment by jcavar** [comment]
Yeah, it works in a bit different way with iOS 11. Large title is part of navigation bar and navigation bar itself stretches when you pull scroll view. `UIRefreshControl` does work though so it should be possible to figure out where it is added and how it works and try to replicate. This will be hacky I feel, if anyone has ideas please let me know
